### ISSUE TITLE
Error in admin interface

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/chat#216]
When going to the admin settings, ownCloud breaks and I get:

```
Fatal error: Call to undefined method OCA\Chat\App\Chat::getContainer(
 in /home/jan/owncloud/apps/chat/lib/admin.php on line 8
```

@LEDfan

**Comment by LEDfan** [comment]
Fixed by d03a173df52b1393aacc9264c696c6e6a97b3d5d

**Comment by jancborchardt** [comment]
Thanks! :)
